src/main/java/io/jenkins/plugins/neuvector/NeuVectorWorker.java

```diff
@@ -640,7 +745,7 @@ private String requestScan(CloseableHttpClient httpclient) throws IOException {
 
         HttpPost httpPostForScan = new HttpPost(uriForScan);
         httpPostForScan.addHeader("Content-Type", "application/json");
-        httpPostForScan.addHeader("X-Auth-Token", token);
+        httpPostForScan.addHeader("X-Auth-Apikey", config.getServerApiKey().getPlainText());
```

Comment:
Does it mean the plugin no longer supports calling POST(/v1/scan/repository) by jwt token?

Reply:
I believe the plugin does not currently support calling POST(/v1/scan/repository) using a JWT token directly.on.

For the scenario where a customer specifies controllerUser/controllerPassword but not apiKey, it will not work as intended because the headers used for authentication explicitly rely on X-Auth-Apikey.

I will try to find a way to use token or apikeys